src/ui/ExpandableSection/ExpandableSection.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import { Meta, StoryObj } from '@storybook/react'
+import React, { useState } from 'react'
+
+import { ExpandableSection } from './ExpandableSection'
+
+const meta: Meta<typeof ExpandableSection> = {
+  title: 'Components/ExpandableSection',
+  component: ExpandableSection,
+}
+export default meta
+
+type Story = StoryObj<typeof ExpandableSection>
+
+const DefaultStory: React.FC = () => {
+  const [isExpanded, setIsExpanded] = useState(false)
+  return (
+    <ExpandableSection>
+      <ExpandableSection.Trigger
+        isExpanded={isExpanded}
+        onClick={() => setIsExpanded(!isExpanded)}
+      >
+        Expandable Section
+      </ExpandableSection.Trigger>
+      <ExpandableSection.Content>
+        This is the content of the expandable section.
+      </ExpandableSection.Content>
+    </ExpandableSection>
+  )
+}
+
+const WithHtmlContentStory: React.FC = () => {
+  const [isExpanded, setIsExpanded] = useState(false)
+  return (
+    <ExpandableSection>
+      <ExpandableSection.Trigger
+        isExpanded={isExpanded}
+        onClick={() => setIsExpanded(!isExpanded)}
+      >
+        Expandable Section with HTML
+      </ExpandableSection.Trigger>
+      <ExpandableSection.Content>
+        <div>
```

Comment:
I think `ExpandableSection.Content` should functionally be a div, so could probably remove this `div` here. Doesn't really matter though